lib/matplotlib/axes/_base.py

```diff
@@ -1780,28 +1780,20 @@ def set_anchor(self, anchor, share=False):
 
         Parameters
         ----------
-        anchor : 2-tuple of floats or {'C', 'SW', 'S', 'SE', ...}
-            The anchor position may be either:
-
-            - a sequence (*cx*, *cy*). *cx* and *cy* may range from 0
-              to 1, where 0 is left or bottom and 1 is right or top.
-
-            - a string using cardinal directions as abbreviation:
-
-              - 'C' for centered
-              - 'S' (south) for bottom-center
-              - 'SW' (south west) for bottom-left
-              - etc.
-
-              Here is an overview of the possible positions:
-
-              +------+------+------+
-              | 'NW' | 'N'  | 'NE' |
-              +------+------+------+
-              | 'W'  | 'C'  | 'E'  |
-              +------+------+------+
-              | 'SW' | 'S'  | 'SE' |
-              +------+------+------+
+        anchor : 2-tuple of floats or {'C', 'SW', 'S', 'SE', 'E', 'NE', ...}
+            Either an (*x*, *y*) pair of relative coordinates (0 is left or
```

Comment:
The effect of the relative coordinates is not so easy to understand. I was mixing this up with `loc` and `bbox_to_anchor` in legends, but that works differently.

Would it be helpful to extend below table by the respective coordinates? e.g. `'NW' (0, 1)` , `'C' (0.5, 0.5)` and so on.

Reply:
good idea